addons/website_slides/controllers/main.py

```diff
@@ -676,6 +676,73 @@ def slide_send_share_email(self, slide_id, email, fullscreen=False):
     # QUIZZ SECTION
     # --------------------------------------------------
 
+    @http.route('/slides/slide/quiz/create', type='json', methods=['POST'], auth='user', website=True)
+    def slide_quiz_create(self, **post):
+        request.env['slide.slide.partner'].search([
+            ('slide_id', '=', post['slide_id']),
+            ('partner_id', '=', request.env.user.partner_id.id)
+        ]).write({'completed': False})
+        question = request.env['slide.question'].create({
+            'sequence': post['sequence'],
+            'question': post['question'],
+            'slide_id': post['slide_id'],
+            'answer_ids': [(0, 0, {
+                'sequence': answer['sequence'],
+                'text_value': answer['text_value'],
+                'is_correct': answer['is_correct']
+            }) for answer in post['answer_ids']]
+        })
+        return {
+            'id': question.id,
+            'sequence': question.sequence,
+            'question': question.question,
+            'answer_ids': [{
+                'id': answer['id'],
+                'text_value': answer['text_value'],
+                'is_correct': answer['is_correct']
+            } for answer in question.answer_ids]
+        }
+
+    @http.route('/slides/slide/quiz/update', type='json', methods=['POST'], auth='user', website=True)
+    def slide_quiz_update(self, **post):
+        question = request.env['slide.question'].browse(post['id'])
```

Comment:
you should check if the question exists first, just in case someone calls this route and send crap to it.